### PR TITLE
Create ui-c8y-1019-23-5-direct-sub-users-count-indicator-for-a-user-is-missing-in.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59054](https://cumulocity.atlassian.net/browse/MTM-59054)
Original PR: [6235](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6235)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1019-23-5-direct-sub-users-count-indicator-for-a-user-is-missing-in.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Direct-sub-users-count-indicator-for-a-user-is-missing-in-users-list-page (#6222) [GRAFT][release/cd] (#6235)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-23-5-direct-sub-users-count-indicator-for-a-user-is-missing-in.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Direct sub-users count indicator for a user is missing in users list page
```

File: content/change-logs/platform-services/ui-c8y-1019-23-5-direct-sub-users-count-indicator-for-a-user-is-missing-in.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59054
+version: 1019.23.5
+---
+Direct-sub-users-count-indicator-for-a-user-is-missing-in-users-list-page (#6222) [GRAFT][release/cd] (#6235)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-23-5-direct-sub-users-count-indicator-for-a-user-is-missing-in.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the users list page, the direct sub-users count indicator for a user was missing. This indicator has now been added to provide visibility into the number of direct sub-users for each user. With this change, administrators and users can easily see the count of direct sub-users, enabling them to better understand and manage the user hierarchy and relationships within the system.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by pink-mongoose — CHANGES_REQUESTED** [review]
Apologies, this is confusing as the file changes only show the title, not the release notes.  But the release notes need improving.  suggest:

In the users list page, the direct sub-users count indicator for a user was missing. This indicator has now been added to provide visibility into the number of direct sub-users for each user.

**Review by Airdroplet — APPROVED** [review]

**Review by pink-mongoose — APPROVED** [review]

**Pull request merged**